packages/docs/src/components/DocsCode.vue

```diff
@@ -18,7 +19,7 @@ export default {
     code: {
```

Comment:
Move `watch` under `props`.

Follow this order:
```
Props
Data
Computed
Methods
Watch
```

Reply:
👍